command/certificate/needsRenewal.go

```diff
@@ -0,0 +1,135 @@
+package certificate
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"io/ioutil"
+	"os"
+	"time"
+)
+
+func needsRenewalCommand() cli.Command {
+	return cli.Command{
+		Name:      "needs-renewal",
+		Action:    cli.ActionFunc(needsRenewalAction),
+		Usage:     `Check if a certificate needs to be renewed`,
+		UsageText: `**step certificate needs-renewal** <crt_file> or <host_name> [**--expires-in <duration>]`,
```

Comment:
We don't really do this `<crt_file> or <host_name>` in other places. Which is not to say that we shouldn't be doing that. 

But I think instead let's try `<crt_file or host_name>` to make it clear that's it’s one argument.

Reply:
@dopey are we naming this needs-renewal instead of verdancy?